.github/workflows/CI_build.yml

```diff
@@ -65,8 +78,6 @@ jobs:
       - name: Install dependencies
         run: |
           python3 -m pip install --upgrade pip setuptools wheel
-          python3 -m pip install -e .[dev]
+          python3 -m pip install -e ".[dev,tensorflow]"
       - name: Run unit tests
-        run: pytest -v
-      - name: Verify that we can build the package
-        run: python3 setup.py sdist bdist_wheel
+        run: coverage run --source=mcfly --data-file=.coverage.tensorflow -m pytest -v
```

Comment:
I don't see why you changed this line. It seems to be doing mostly the same thing, except for logging the coverage as well now. These results are however never used. Fine with me, but curious why you changed this.